package.json

```diff
@@ -41,6 +41,7 @@
   },
   "dependencies": {
     "bcryptjs": "2.4.3",
+    "bluebird": "3.5.0",
```

Comment:
Any particular reason you need this?

Reply:
I'm using Promise.each & Promise.map which are quite handy.
Do you have alternatives?